data/mods/DinoMod/monsters/zed-dinosaur.json

```
@@ -848,72 +949,12 @@
     "type": "MONSTER",
     "id": "mon_zalamosaurus_juvenile",
     "name": { "str": "juvenile Alamosaurus zombie" },
-    "copy-from": "mon_zapatosaurus",
+    "copy-from": "mon_zapatosaurus_juvenile",
     "looks_like": "mon_apatosaurus",
     "armor_bash": 7,
     "armor_cut": 9,
     "armor_bullet": 1,
     "description": "A small four-legged plant-eating dinosaur juvenile corpse with natural bone armor plates.",
```

```suggestion
    "description": "The corpse of a small, four-legged, herbivorous dinosaur juvenile, with natural bone armor plates.",
```
Not sure if this is quite right, but it's better IMO.